README.md

```diff
@@ -431,7 +431,8 @@ A number of options are available to allow you to customise the SDK:
 
   - `showCountrySelection` (boolean - default: `false`)
 
-    If only one document type is preselected that is not a passport document, the issuing country selection screen can still be displayed by setting this to `true`.
+    The country selection screen is never displayed when a passport document is selected.
```

Comment:
I would suggest highlighting the fact that showCountrySelection controls what happens when a single document is being selected, and that it has no effect when multiple documents are being selected in this paragraph instead of hiding it in the examples below. The examples are still good, but I think this needs to be the first sentence in the description of the config.